doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -253,6 +253,10 @@ nRF Desktop
     The value is now aligned with the Fast Pair requirements.
   * The :kconfig:option:`CONFIG_NRF_RRAM_WRITE_BUFFER_SIZE` Kconfig option value in the nRF54L15 PDK configurations to ensure short write slots.
     It prevents timeouts in the MPSL flash synchronization caused by allocating long write slots while maintaining a Bluetooth LE connection with short intervals and no connection latency.
+  * The method of obtaining hardware ID using Zephyr's :ref:`zephyr:hwinfo_api` on the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```

Comment:
```suggestion
  * The method of obtaining hardware ID using Zephyr's :ref:`zephyr:hwinfo_api` on the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```
Shouldn't the nRF54H20 DK be linked to the NCS page here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.html#ug-nrf54h20-gs.
For example, like in here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.html#enhanced_shockburst_esb
@FrancescoSer - Could you confirm this. I could see both examples in the release notes - linking to zephyr page as well as to the NCS page.

Reply:
discussed offline with Francesco, and since the zephyr page is the board page, we often link to that when listing DKs and boards.

Reply:
thanks @divipillai for this clarification :)